e2e/cypress/integration/integrations/outgoing_webhook/search_on_outoing_webhooks_spec.js

```diff
@@ -0,0 +1,118 @@
+    const Charlie = 'Charlie Common';
+    const triggerA = 'apple';
+    const triggerB = 'banana';
+    const triggerC = 'carrot';
+
+    before(() => {
+        const callbackUrl = `${Cypress.env().webhookBaseUrl}/post_outgoing_webhook`;
+
+        cy.requireWebhookServer();
+
+        // # Create test team and 3 outgoing web-hooks
+        cy.apiInitSetup().then(({team}) => {
+            testTeam = team.name;
+
+            const firstOutgoingHook = {
+                team_id: team.id,
+                display_name: Alpha,
+                trigger_words: [triggerA],
+                callback_urls: [callbackUrl],
+            };
+
+            const secondOutgoingHook = {
+                team_id: team.id,
+                display_name: Bravo,
+                trigger_words: [triggerB],
+                callback_urls: [callbackUrl],
+            };
+
+            const thirdOutgoingHook = {
+                team_id: team.id,
+                display_name: Charlie,
+                trigger_words: [triggerC],
+                callback_urls: [callbackUrl],
+            };
+
+            cy.apiCreateWebhook(firstOutgoingHook, false).then((hook) => {
+                outgoingWebhook = hook;
+
+                cy.apiGetOutgoingWebhook(outgoingWebhook.id).then(({webhook, status}) => {
+                    expect(status).equal(200);
+                    expect(webhook.id).equal(outgoingWebhook.id);
+                });
+            });
+
+            cy.apiCreateWebhook(secondOutgoingHook, false).then((hook) => {
+                outgoingWebhook = hook;
+
+                cy.apiGetOutgoingWebhook(outgoingWebhook.id).then(({webhook, status}) => {
+                    expect(status).equal(200);
+                    expect(webhook.id).equal(outgoingWebhook.id);
+                });
+            });
+
+            cy.apiCreateWebhook(thirdOutgoingHook, false).then((hook) => {
+                outgoingWebhook = hook;
+
+                cy.apiGetOutgoingWebhook(outgoingWebhook.id).then(({webhook, status}) => {
+                    expect(status).equal(200);
+                    expect(webhook.id).equal(outgoingWebhook.id);
+                });
+            });
+        });
+    });
+
+    it('MM-T614 Search on Outgoing Webhooks page', () => {
+        // * Assert that search for Alpha (lower-case) returns only Alpha webhook
+        cy.visit(`/${testTeam}/integrations/outgoing_webhooks`);
+        cy.get('#searchInput').type('alpha').then(() => {
+            cy.get('.backstage-list__item').should('not.contain', triggerB).should('not.contain', triggerC).contains(triggerA);
+        });
+
+        // * Assert that search for Bravo (upper-case) returns only Bravo webhook
+        cy.visit(`/${testTeam}/integrations/outgoing_webhooks`);
+        cy.get('#searchInput').type('BRAVO').then(() => {
+            cy.get('.backstage-list__item').should('not.contain', triggerA).should('not.contain', triggerC).contains(triggerB);
+        });
```

Comment:
Revisiting the page multiple times will make the test longer. I suggest to just clear the input then type again.
```suggestion
        cy.get('#searchInput').clear().type('BRAVO');
        cy.get('.backstage-list__item').should('not.contain', triggerA).should('not.contain', triggerC).contains(triggerB);
```

Same comment below.